internal/serviceconfig/serviceconfig_test.go

```diff
@@ -133,3 +135,57 @@ func TestBalancerConfigUnmarshalJSON(t *testing.T) {
 		})
 	}
 }
+
+func TestBalancerConfigMarshalJSON(t *testing.T) {
+	tests := []struct {
+		name     string
+		bc       BalancerConfig
+		wantJSON string
+	}{
+		{
+			name: "OK",
+			bc: BalancerConfig{
+				Name:   testBalancerBuilderName,
+				Config: testBalancerConfig,
+			},
+			wantJSON: fmt.Sprintf("[{%q: %v}]", testBalancerBuilderName, testBalancerConfigJSON),
```

Comment:
Would it make more sense to set `wantJSON` to the string literal of the actual expected JSON config? That would be a little more readable I feel.

Reply:
Done.